manifests/k/ksnip/ksnip/Continuous/1.10.0/ksnip.ksnip.Continuous.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: ksnip.ksnip.Continuous
+PackageVersion: 1.10.0
+PackageLocale: en-US
+Publisher: ksnip
+PublisherUrl: http://ksnip.org/
+PublisherSupportUrl: https://github.com/ksnip/ksnip/issues
+# PrivacyUrl: 
+Author: Damir Porobic
+PackageName: ksnip
```

I believe this will cause a conflict with ksnip.ksnip so we have to add (Continuous) so it only uses the ProductCode for the matching
I believe this is where Channels come into play once we get that working.
```suggestion
PackageName: ksnip (Continuous)
```

---

Thanks, forgot about that